ufc/bandit-tests/test-case-bandit-does-not-exist.json

```diff
@@ -0,0 +1,12 @@
+{
+    "flag": "flag_does_not_exist",
+    "defaultValue": "default",
+    "subjects": [
+      {
+        "subjectKey": "alice",
+        "subjectAttributes": {"numeric_attributes": {"age": 25}, "categorical_attributes": {"country": "USA"}},
+        "actions": [],
+        "assignment": {"variation": "default", "action": null}
```

Comment:
Having an assignment with a variation and action component is breaking from our composite (dynamic variant value) approach. Happy to proceed this way--as its a step towards detailed return objects--but it's a pretty big change.